### PR TITLE
meta: add build order template to PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by OriginCode**
This PR updates the current pull request template for aosc-os-abbs topic-based packaging procedure.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -29,7 +29,12 @@ Security Update?
 Build Order
 -----------
 
-Please describe in what order maintainers should build this pull request.
+<!-- Please describe in what order maintainers should build this pull request. You can use the following template. -->
+
+```bash
+ciel build <PACKAGES>
```
Comment (MingcongBai) [inline .github/PULL_REQUEST_TEMPLATE]: Consider removing `ciel build` form the sequence, as it denotes the command, not the order in which packages are built. It also adds unnecessary clutter in this single-line format.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (OriginCode) [reply .github/PULL_REQUEST_TEMPLATE]: Removed.

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -29,7 +29,12 @@ Security Update?
 Build Order
 -----------
 
-Please describe in what order maintainers should build this pull request.
+<!-- Please describe in what order maintainers should build this pull request. You can use the following template, use space to separate packages. -->
+
+```
+PACKAGES...
```
Comment (MingcongBai) [inline .github/PULL_REQUEST_TEMPLATE]: May be better denoted by `pkg1 pkg2 pkg3 ...`.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (OriginCode) [reply .github/PULL_REQUEST_TEMPLATE]: Done

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**